gatsby/content/blog/2023/02/2023-02-16-synapse-1.77.mdx

```diff
@@ -0,0 +1,46 @@
+---
+date: '2023-02-16'
+title: Synapse 1.77 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+Greetings Matrix fans! We've published Synapse [version 1.77](https://github.com/matrix-org/synapse/releases/tag/v1.77.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience.
+
+## The new stuff
+
+### Experimental support for intentional mentions (MSC3952)
+
+Mentioning other users on Matrix is difficult: it is not possible to know if mentioning a user
+by display name or Matrix ID will count as a mention, but is also too easy to mistakenly mention a user.
+
+[MSC3952](https://github.com/matrix-org/matrix-spec-proposals/pull/3952) is a way to solve that trouble
+by making the mentioning explicit in the protocol using a dedicated event property,
+instead of relying on searching the body of the message as before.
+
+Synapse now implements this as an experimental feature.
+
+### Experimental support to suppress notifications from message edits (MSC3958)
+
+Have you already been annoyed by a noisy notification that comes back, but you can't really pinpoint
+from where this is coming ?
+This is usually because currently, editing a message where you are mentioned (or including an `@room`) will trigger
+a noisy notification, which can be pretty annoying when the message is edited 10 times.
```

Comment:
```suggestion
This is usually because edits to a message where you are mentioned (or that mention `@room`) will trigger
retrigger a noisy notification.  That can be pretty annoying when the message is edited 10 times!
```